socorro/external/boto/connection_context.py

```diff
@@ -202,6 +206,9 @@ def submit(self, id, name_of_thing, thing):
         key_object = bucket.new_key(key)
         key_object.set_contents_from_string(thing)
 
+        elapsed_time = time.time() - start_time
+        self.metrics.histogram(name_of_thing, value=elapsed_time * 1000.0)
```

Comment:
As currently written, if any of the functions in between start_time and elapsed_time throw an exception, we won't record the histogram.

It would be good to send it in both cases, like we do for elasticsearch.